test/node.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var assert = require('simple-assert');
+var type = require('..');
+(function () {
+  if (typeof process === 'undefined' || typeof process.release !== 'object' || !process.release.name) {
+    return describe.skip;
+  }
+  return describe;
+}())('Node Specific', function () {
```

Comment:
Yeah, good call. I actually wrote the Node tests before adding describeIf to other tests, and haven't touched them since - but I've changed it now.